e2e-tests/IIoTPlatform-E2E-Tests/Standalone/A_PublishSingleNodeStandaloneDirectMethodTestTheory.cs

```diff
@@ -117,6 +126,15 @@ IIoTMultipleNodesTestContext context
             nodesOnEndpoint.EndpointUrl = request.EndpointUrl;
             var requestGetConfiguredNodesOnEndpoint = nodesOnEndpoint.ToApiModel();
 
+            //Call GetConfiguredEndpoints direct method
+            responseGetConfiguredEndpoints = await TestHelper.CallMethodAsync(_iotHubClient, _iotHubPublisherDeviceName, _iotHubPublisherModuleName, new MethodParameterModel {
+                Name = TestConstants.DirectMethodNames.GetConfiguredEndpoints
+            }, _context, cts.Token).ConfigureAwait(false);
+
+            Assert.Equal((int)HttpStatusCode.OK, responseGetConfiguredEndpoints.Status);
+            configuredEndpointsResponse = _serializer.Deserialize<List<PublishNodesEndpointApiModel>>(responseGetConfiguredEndpoints.JsonPayload);
+            Assert.Equal(configuredEndpointsResponse.Count, 1);
```

Comment:
Please validate that all fields of returned endpoint are as expected.

Reply:
The sample request has only EndpointUrl; all other fields are null. Will check accordingly.